doc/source/conf.py

```diff
@@ -214,7 +214,7 @@ def check_sphinx_version(expected_version):
 
 # The theme to use for HTML and HTML Help pages.  Major themes that come with
 # Sphinx are currently 'default' and 'sphinxdoc'.
-html_theme = 'stsci_rtd_theme'
+html_theme = 'sphinx_rtd_theme'
```

Comment:
You should also remove the 'import' statement for this theme from the top of the conf.py file.

Reply:
Issue resolved in commit [17b6221](https://github.com/spacetelescope/drizzlepac/pull/935/commits/17b622142f369a9c9b0ef1ac77a63cac4742826f)